vendor/broccoli-fontawesome-pack.js

```diff
@@ -46,12 +47,19 @@ FontAwesomePack.prototype.build = function() {
   if(this.options.icons === 'all'){
     selectedIcons = Object.keys(pack[pack.prefix])
   } else {
-    selectedIcons = this.options.icons
+    selectedIcons = this.options.icons.map(iconName => {
+      const prefix = config.familyPrefix;
+      if (iconName.substr(0, 2) === prefix) {
```

Comment:
Ok, an issue with this is going to be [`fax`](https://fontawesome.com/icons/fax?style=solid). I think that icon will fail to resolve properly.